src/languageProvider/locales/en_US.json

```diff
@@ -105,12 +105,14 @@
   "str.anytime": "anytime",
   "str.back": "Back",
   "str.bet": "Bet",
+  "str.betRound": "Betting Round",
```

Comment:
lets use the full name for small enough strings: `str.bettingRound`. so the string is self explanatory.